### PR TITLE
style: :wrench: add navbar headers, like style, etc.

### LINKED ISSUE #83: Create navbar headers with connected files to split into style, software, and data architecture

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

- This PR adds navbar headers for the different sections, plus renames the data architecture folder to match the other folders.

## Related Issues

Closes #83

## Reviewer Focus

This PR only needs a quick review.

## Checklist

- [x] Spell-check
- [x] Did the page(s) preview correctly on your machine without breaking

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -64,4 +64,4 @@ written two core parts in mind:
 -   A [software architecture](software/index.qmd) document, which is
     based on the [arc42](https://arc42.org) template and the [C4
     Model](https://c4model.com/)
--   A data architecture document, which is a work-in-progress
\ No newline at end of file
+-   A data architecture document, which is a work-in-progress
```
Comment (signekb) [inline index.qmd:67]:
```suggestion
-   A [data architecture](data/index.qmd) document, which is a work-in-progress
```

File: data/index.qmd [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Data architecture
 
+{{< include /includes/_wip.qmd >}}
+
```
Comment (signekb) [inline data/index.qmd:4]: I added a wip to this post :)

**Review by signekb — CHANGES_REQUESTED** [review]
Looks good! 🌼 I added a wip to `data/index.qmd` and a suggestion to link to the page in `index.qmd`

**Review by lwjohnst86 — COMMENTED** [review]
@signekb updated!

**Review by signekb — APPROVED** [review]
Fantastic! 🎉

**Pull request merged**